cores/nRF5/main.cpp

```diff
@@ -24,8 +24,8 @@ void Bluefruit_printInfo() __attribute__((weak));
 void Bluefruit_printInfo() {}
 #endif
 
-// DEBUG Level 3
-#if CFG_DEBUG >= 3
+// From the UI, setting debug level to 3 will enable SysView
```

Comment:
the comment text is not relevant anymore, you can safely remove it

Reply:
I think this comment is very accurate.  When the debug level is set to 3, SysView is enabled.  This is the old behavior, and continues to be the behavior with this PR.  So, I am not sure I understand why this would not be a relevant comment?